buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -113,6 +118,9 @@
         DEFAULT_CACHE_TRIM_INTERVAL = SystemPropertyUtil.getInt(
                 "io.netty.allocator.cacheTrimInterval", 8192);
 
+        DEFAULT_CACHE_TRIM_RATE = SystemPropertyUtil.getLong(
+                "io.netty.allocator.cacheTrimRateMs", 0);
```

Comment:
How about `io.netty.allocation.cacheTrimIntervalMillis`? To avoid confusion, we could deprecate `cacheTrimInterval` in favor of `cacheTrimIntervalCount`. The current name `rate` is misleading because it's not really a rate.

Reply:
@trustin I think I would like to still preserve the old one as well. Let me rename it tho.